### PR TITLE
Link table

### PR DESCRIPTION
**Pull request by touredri**
In this PR we used join to link table and show based on different criteria 

Capture d’écran 2023-06-09 093336
Capture d’écran 2023-06-09 093416
Capture d’écran 2023-06-09 093750
Capture d’écran 2023-06-09 093840
Capture d’écran 2023-06-09 093917
Capture d’écran 2023-06-09 094655
Capture d’écran 2023-06-09 094802
Capture d’écran 2023-06-09 191831
Capture d’écran 2023-06-09 193312

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @touredri

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly remove all comments that are there to guide you to fill in the readme template.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -29,10 +29,17 @@ ALTER TABLE animals ADD COLUMN owners_id INT,
     ADD FOREIGN KEY (owners_id) REFERENCES owners(id);
```
Comment (tugsem) [inline schema.sql:31]: - [ ] Kindly add the table for "vets" as per the requirement: 
image

File: data.sql [hunk 2]
```diff
@@ -21,3 +21,189 @@ update animals set owners_id = (select id from owners where full_name = 'Jennife
 update animals set owners_id = (select id from owners where full_name = 'Bob')  where name = 'Devimon' or name = 'Plantmon'; 
 update animals set owners_id = (select id from owners where full_name = 'Melody Pond')  where name = 'Charmander' or name = 'Squirtle' or name= 'Blossom';
 update animals set owners_id = (select id from owners where full_name = 'Dean Winchester')  where name = 'Angemon' or name = 'Boarmon';
+
+
```
Comment (tugsem) [inline data.sql:31]: - [ ] Kindly add these data to your newly created "vets" table as per the requirement:
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @touredri 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️ visits and specializations tables are created
✔️ The required data is inserted for visits and specialization tables
## Required Changes ♻️
_Check the comments under the review._

- [ ] Please remove all the comments from your readme.md file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (touredri) [reply data.sql:31]: I forgot @tugsem

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @touredri 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Understand the different types of relationships between tables.
- ✔️ Prepare complex queries that answer analytical questions.
- ✔️ Use primary key & foreign key mechanisms for joining tables.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**